pkg/vmprovider/providers/vsphere2/vmlifecycle/bootstrap.go

```diff
@@ -61,20 +61,25 @@ func DoBootstrap(
 	bootstrap := vmCtx.VM.Spec.Bootstrap
 	if bootstrap == nil {
 		// For backwards compat we have to keep going and default to LinuxPrep.
-		bootstrap = &vmopv1.VirtualMachineBootstrapSpec{}
+		bootstrap = &vmopv1.VirtualMachineBootstrapSpec{
+			LinuxPrep: &vmopv1.VirtualMachineBootstrapLinuxPrepSpec{
+				HardwareClockIsUTC: true,
+			},
+		}
 	}
 
 	cloudInit := bootstrap.CloudInit
 	linuxPrep := bootstrap.LinuxPrep
-	sysprep := bootstrap.Sysprep
+	sysPrep := bootstrap.Sysprep
 	vAppConfig := bootstrap.VAppConfig
 
-	bootstrapArgs, err := getBootstrapArgs(vmCtx, k8sClient, cloudInit != nil, networkResults, bootstrapData)
+	isGOSC := linuxPrep != nil || sysPrep != nil
```

Comment:
I left a comment later about this flag. I'm not certain we need it -- it may be an over-optimization.